src/main/webapp/sass/_core.scss

```diff
@@ -441,6 +441,11 @@ end sticky footer
 }
 
 .downloadLink,.downloadLink:hover,.downloadLink:visited { text-decoration:underline; margin-left:5px;}
+#body .Download-Link {
```

Comment:
Typically set to the primary color of the portal, but our header background is current that color, so we need to override that style (to make the color white).